components/private/ThumbTrack/src/MDCThumbTrack.h

```diff
@@ -26,21 +26,46 @@
 /** The delegate for the thumb track. */
 @property(nullable, nonatomic, weak) id<MDCThumbTrackDelegate> delegate;
 
-/** The color of the thumb and left track. */
-@property(nullable, nonatomic, strong) UIColor *primaryColor;
-
-/** The color of the thumb off color. */
-@property(nullable, nonatomic, strong) UIColor *thumbOffColor;
```

Comment:
Sanity check: do we need to deprecate this API because of any internal client usage?

Reply:
There is one internal client that we can migrate (a single usage) to the new API.

Reply:
Please keep the existing API around then and perform the migrations post-release.

Reply:
No internal clients are using `thumbOffColor` (and neither were we).  Do you still want to keep it?